src/mono/sample/wasm/browser-nextjs/components/deepThought.js

```diff
@@ -1,29 +1,29 @@
 import { useState, useEffect } from 'react'
-import createDotnetRuntime from '@microsoft/dotnet-runtime'
+import { dotnet } from '@microsoft/dotnet-runtime'
 
 let dotnetRuntimePromise = undefined;
-let meaningFunction = undefined;
 
 async function createRuntime() {
     try {
         const response = await fetch('dotnet.wasm');
         const arrayBuffer = await response.arrayBuffer();
-        return createDotnetRuntime({
-            configSrc: "./mono-config.json",
-            disableDotnet6Compatibility: true,
-            locateFile: (path, prefix) => {
-                return '/' + path;
-            },
-            instantiateWasm: async (imports, successCallback) => {
-                try {
-                    const arrayBufferResult = await WebAssembly.instantiate(arrayBuffer, imports);
-                    successCallback(arrayBufferResult.instance);
-                } catch (err) {
-                    console.error(err);
-                    throw err;
+        
+        return dotnet.
+            withModuleConfig({
+                locateFile: (path, prefix) => {
```

Comment:
perhpas default locateFile now works better and it's not necessary anymore ?

Reply:
In nextjs `import.meta.url` points to `file:///...`. I tried to update webpack, but that's not it.